CourseApp/PigAndSalo.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace CourseApp
+{
+    public class PigAndSalo
+    {
+        private string pig;
+        private string salo;
+        private int age;
+        private int dateOfDeath;
+
+        public PigAndSalo(string pig, string salo, string color, int age, int dateOfDeath)
+        {
+            Pig = pig;
+            Salo = salo;
+            Age = age;
+            DateOfDeath = dateOfDeath;
+        }
+
+        public string Pig
```

Comment:
Pig here sinonym to the name? if it does not do anything in get and set - then simply use Pig {get; set;}

Reply:
@Nikrybkin please fix this one